db/sql/0_msar.sql

```diff
@@ -126,6 +136,20 @@ END;
 $$ LANGUAGE plpgsql RETURNS NULL ON NULL INPUT;
 
 
+CREATE OR REPLACE FUNCTION
+__msar.get_fully_qualified_object_name(sch_name text, obj_name text) RETURNS text AS $$/*
+Return the fully-qualified name for a given database object (e.g., table).
+
+Args:
+  sch_name: The schema of the object, quoted.
+  obj_name: The name of the object, unqualified and quoted.
+*/
+BEGIN
+  RETURN format('%s.%s', sch_name, obj_name);
+END;
+$$ LANGUAGE plpgsql RETURNS NULL ON NULL INPUT;
```

Comment:
Is this function still necessary, given that it's only called in the function immediately below? I think it would be okay to combine the two.

Reply:
It's not necessary, but doesn't it make sense to have this function be available in the private and public contexts? Considering that the parameter quoting requirements are different?